app/views/account/AccountSettingsView.coffee

```diff
@@ -131,7 +131,7 @@ module.exports = class AccountSettingsView extends CocoView
           window?.webkit?.messageHandlers?.notification?.postMessage(name: "signOut") if window.application.isIPadApp
           Backbone.Mediator.publish("auth:logging-out", {})
           window.tracker?.trackEvent 'Log Out', category:'Homepage' if @id is 'home-view'
-          logoutUser($('#login-email').val())
+          logoutUser()
```

Comment:
Why this change?

Reply:
This argument was never used in the legacy code path and looks safe to remove.
I feel safer removing it as it was last changed 5 years ago.